app-modules/etno/src/Filament/Actions/Items/OrderFilesByNameAction.php

```
@@ -0,0 +1,50 @@
+    {
+        parent::setUp();
+
+        $this->label('Order by Name')
+            ->icon(Heroicon::BarsArrowDown)
+            ->hidden(fn (?array $state): bool => \count($state ?? []) < 2)
+            ->action(function (Repeater $component) {
+                $state = $component->getState();
+                uasort($state, function ($a, $b) {
+                    $path = $this->getFileStatePath();
+
+                    $fileA = $path ? Arr::get($a, $path) : $a;
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Illuminate\Support\Arr' in method 'setUp'. (undefined)

(StaticAccess)